ml/pre-processing/get_VT_signatures.scala

```diff
@@ -0,0 +1,45 @@
+import com.datastax.spark.connector._
+import play.api.libs.json.Json
+import java.io.{ByteArrayOutputStream, ByteArrayInputStream}
+import java.util.zip.{GZIPOutputStream, GZIPInputStream}
+
+case class VT_results_by_service_name_class(service_name: String, sha256: String)
+case class VT_results_by_sha256_class(sha256: String, service_name: String, results: Array[Byte] )
+case class VT_join_results_class(sha256: String, service_name: String, results: String)
+case class VT_sample_signatures_initial_seq_rdd_class(sha256: String, seq_results: Seq[String])
+case class VT_sample_signatures_final_array_rdd_class(sha256:String, array_results:Array[Double])
+
+def unzip(x: Array[Byte]) : String = {		
+    val inputStream = new GZIPInputStream(new ByteArrayInputStream(x))
+    val output = scala.io.Source.fromInputStream(inputStream).mkString
+    return output
+}
+def deleteNumberInSampleSignatures(x: String): Boolean = {
+    val regex = "[0-9]".r
+    return regex.findFirstIn(x).isEmpty
+}
+
+val VT_results_by_service_name_meta = sc.cassandraTable[VT_results_by_service_name_class]("gsoc3","results_meta_by_service_name").where("service_name=?","virustotal")
```

Comment:
Yes, I plan to have a configuration file. But I failed.
The code is this:
```
import java.io.BufferedInputStream
import java.io.File
import java.io.FileInputStream
import java.io.InputStream
import org.apache.hadoop.conf._
import org.apache.hadoop.fs._
val conf = new Configuration()
val fileSystem = FileSystem.get(conf)
val stream = fileSystem.open(new Path("Configuration_File.txt"))
def readLines = Stream.cons(stream.readLine, Stream.continually( stream.readLine))
readLines.takeWhile(_ != null).foreach(line => println(line))
```
I plan to read configuration file by scala readfile method. However, there is this line in the code.
```
val conf = new Configuration()
```
Because I new the Configuration(), the spark interpreter configuration is covered and then the spark works with error. 
I try to solve this issue by getting spark configuration instead of newing configuration. But I cannot google the answer.